src/lib/web-component/log-view/log-view-element.js

```diff
@@ -64,6 +75,11 @@ export class LogView extends HTMLElement {
       rowsFragment.appendChild(logTableRowInstance);
     }
     this.tableBody.appendChild(rowsFragment);
+
+    this.totalLogs += logs.length;
+    this.dispatchEvent(
+      new CustomEvent('logcounter', { detail: { totalLogs: this.totalLogs } })
```

Comment:
This logic is incorrect, you also need to emit the number of filtered logs, as that may change by the outcome of `logTableRowInstance.hidden = !this.isFilterMatched(log);` above.

Reply:
aren't we showing the total number of logs that are collected in the background here?

Reply:
We are showing the total number of logs, whether it's from the background or a file doesn't matter.

Reply:
I made another method named `updateFilteredRowsCount` which gets called when filters are applied OR when new logs are rendered.